lib/jellyfish/component/recording.ex

```diff
@@ -23,6 +23,7 @@ defmodule Jellyfish.Component.Recording do
         """)
 
     with {:ok, serialized_opts} <- serialize_options(options, Options.schema()),
+         result_opts <- Map.update!(serialized_opts, :subscribe_mode, &String.to_atom/1),
```

Comment:
Suggestion: I would wrap this line in function `get_subscribe_mode` or `parse_subscribe_mode` for consistency